requirements.txt

```diff
@@ -6,6 +6,7 @@ django-cors-middleware==1.3.1
 django-test-without-migrations==0.4
 djangorestframework==3.5.3
 gunicorn==19.6.0
+pandas==0.19.2
```

Comment:
Hum… if you're using Anaconda we have to change the documentation. But… before we reach this point: I don't think it's a good idea to proceed with this issue and PR. The overhead of requiring to Anaconda is too big to save a couple of minutes in the seed process.